.travis.yml

```diff
@@ -17,12 +17,10 @@ jdk:
   - oraclejdk8
 
 install:
-  - mvn clean install -DskipTests -Dassembly.skipAssembly=true > /dev/null
+  - bash .travis_install.sh
 
 script:
-  - cd $PINOT_MODULE
-  - cp ../.travis_cmd.sh .
-  - bash .travis_cmd.sh
+  - bash .travis_test.sh
 
 after_success:
   - bash <(curl -s https://codecov.io/bash)
```

Comment:
This could be a security/liability issue if someone compromises the codecov.io script to send spam and/or DDoS some other website. It would be better to check out a known revision of the script (which is available at https://github.com/codecov/codecov-bash) and invokes it as part of a script that does the `after_success` part of the build than fetching an unknown version of it.